config/v1/techpreview.infrastructure.testsuite.yaml

```diff
@@ -12,3 +12,140 @@ tests:
       apiVersion: config.openshift.io/v1
       kind: Infrastructure
       spec: {}
+  onUpdate:
+  # platformStatus tests have to run in "onUpdate" because the Infrastructure has a
+  # status subresource, so we can't create an object with status.
+  - name: Should be able to find the default LoadBalancerType
```

Comment:
Should set LoadBalancerType to OpenShiftManaged if not specified

Reply:
I'll change that, ok.